### DISCUSSION TITLE
Radio3.0@claudiux - New icon

### DISCUSSION THREAD

**Discussion by Hilyxx** [linuxmint/discussions#85] [Ideas]
Radio 3.0 is a great and a feature-rich applet. Unfortunalty png icon does not do justice to it. 

I made a icon with more contrast and more apparent for Cinnamon spices web site. 

Old icon : 
![icon (copie)](https://github.com/linuxmint/discussions/assets/21991448/6c50b5c1-afaa-4e70-bc30-1bcae21d4430)

New icon
![radio](https://github.com/linuxmint/discussions/assets/21991448/6b1f2a35-a155-416c-b948-256d90e81035)

@Claudiux If you are interested, i can give you this new icon

NB : i'm not a professional or designer ^^

**Comment by claudiux** [comment]
@Hilyxx Thanks. Are you the author of this icon?

**Reply by Hilyxx** [reply]
[Radio3-0_icon.zip](https://github.com/linuxmint/discussions/files/12136250/Radio3-0_icon.zip)
Of course. I made it with inkscape. I'm not developper but i like to create icons and themes for linux.

**Reply by claudiux** [reply]
Icon adopted! Many thanks to you!

**Reply by claudiux** [reply]
Can you make them in 128x128, please? That is the limit for Cinnamon.

**Reply by Hilyxx** [reply]
[radio128.svg.zip](https://github.com/linuxmint/discussions/files/12136531/radio128.svg.zip)

**Reply by Hilyxx** [reply]
pas de soucis ça me fait plaisir de contribuer (un peu) :)

**Reply by claudiux** [reply]
You're a great contributor! https://github.com/linuxmint/cinnamon-spices-applets/pull/4966

**Reply by claudiux** [reply]
The new icon is not yet visible on the website. Maybe tomorrow?

**Comment by claudiux** [comment]
I don't understand why the new icon is not yet visible on the website.

**Reply by Hilyxx** [reply]
the new icon is displayed but you need to clear cache and data in web browser
![Capture d’écran du 2023-07-23 14-14-16](https://github.com/linuxmint/discussions/assets/21991448/3d31591b-bda5-4655-b1a7-2458131182bb)

**Comment by Hilyxx** [comment]
Hello @claudiux 

I've updated radio icon for your applet.
- Speaker is more clear and can adapt more easily to various screen resolutions
- fixed a small imperfection.
![icon256](https://github.com/linuxmint/discussions/assets/21991448/7938ea44-ba51-44e8-bcb9-4e569b530305)

[Radio icon.zip](https://github.com/linuxmint/discussions/files/13463758/Radio.icon.zip)

**Comment by claudiux** [comment]
Hello @Hilyxx,

Thanks a lot! Can you make a PR, or would you like me to do it myself?

**Reply by Hilyxx** [reply]
J'ai fait le PR. Je suis novice la dessus donc j'espère que c'est bon.
